src/encoded/types/base.py

```diff
@@ -145,6 +145,9 @@ class Item(SnovaultItem):
         'released': acl.ALLOW_CONSORTIUM_MEMBER_VIEW_ACL,
         # Everyone can view - restricted to specific items via schemas.
         'public': acl.ALLOW_EVERYONE_VIEW_ACL,
+        # Intended to do additional computation to determine if download
+        # is allowed if it's a downloadable file, otherwise same as "released"
+        'restricted': acl.ALLOW_CONSORTIUM_MEMBER_VIEW_ACL,
```

Comment:
This somewhat overlaps with the [existing property](https://github.com/smaht-dac/smaht-portal/blob/main/src/encoded/schemas/mixins.json#L278-L288) for access (that does nothing). Perhaps can turn it into a calculated property based on status now?

Reply:
Not sure I understand this comment?

Reply:
For the benchmarking release, we added a dummy property on files `access_status` that indicates whether the file is designated open or restricted (I set it manually for the currently released files). Now that we have this status, may make sense to replace that property with a calculated property such that certain `status` values are correspond to`open` (such as `public` and `released`) and `restricted` corresponds to `Protected`. Maybe shouldn't implement this now and just keep as a dummy value until the dbgap stuff is in place; not sure on the timing.

Reply:
Oh I see what you mean, I'll make this a calc-prop then as you suggest